app/scripts/controllers/network/network-controller.test.ts

```diff
@@ -3919,7 +3997,7 @@ describe('NetworkController', () => {
             },
             {
               referrer: 'https://test-dapp.com',
-              source: MetaMetricsNetworkEventSource.Dapp,
+              source: 'dapp',
```

Comment:
Q: do we intend to add the constant back in with the core tests after the sync?

Reply:
Yes, I want to do a full audit of these tests at a later point once they are all in `core`, but I think that would be a good change specifically. It certainly is clearer and matches our conventions.